meteor/lib/collections/RundownLayouts.ts

```diff
@@ -46,6 +47,21 @@ export interface RundownLayoutElementBase {
 export interface RundownLayoutExternalFrame extends RundownLayoutElementBase {
 	type: RundownLayoutElementType.EXTERNAL_FRAME
 	url: string
+	scale: number
+}
+
+export enum RundownLayoutMultiViewRole {
+	QUEUE = 'queue',
+	TAKE = 'take',
+	PROGRAM = 'program'
+}
+
+export interface RundownLayoutMultiView extends RundownLayoutElementBase {
+	type: RundownLayoutElementType.MULTIVIEW
+	windowNumber: number
```

Comment:
This property doesnt appear to be read anywhere, its only use is the ui input field.

Should this element be called a multiview? It doesnt look specific to that particularly, and looks to be a more generic name overlay with an action.

Reply:
I've removed windowNumber (https://github.com/olzzon/tv-automation-server-core/commit/6ff7353b14fec1f87f84d11d0a347a480c5b609e), it was a leftover from the initial implementation.

Yeah, it's not specific to that as such. I don't have an alternate name myself, @jstarpl any suggestions?

Reply:
We discussed 'adlib region'. Could you change this too?

Reply:
I've renamed them to AdLib Regions https://github.com/olzzon/tv-automation-server-core/commit/01e0301dc0e2d21ff48d6516a4d11a752532d0a8